gm4_heart_canisters/data/heart_canisters/functions/update.mcfunction

```diff
@@ -0,0 +1,14 @@
+scoreboard players operation @s heart_can_time -= @s heart_can_health
```

Comment:
This function is to vaguely named. "apply_health_boost" or similar would be more descriptive.